src/main/java/de/sakpaas/backend/v2/controller/UserController.java

```diff
@@ -29,6 +46,27 @@ public UserController(UserService userService) {
   @GetMapping("/info")
   public ResponseEntity<UserInfoDto> getUserInfo(@RequestHeader("Authorization") String header,
                                                  Principal principal) {
-    return new ResponseEntity<>(userService.getUserInfo(header, principal), HttpStatus.OK);
+    return new ResponseEntity<>(userService.getUserInfo(header, principal), OK);
+  }
+
+  /**
+   * Get Endpoint that returns the favorites of the user specified in the token.
+   *
+   * @param header The Authorization-Header that has to be provided in the request.
+   * @return Returns an Array of Locations.
+   */
+  @GetMapping("/self/favorites")
+  public ResponseEntity<List<LocationResultLocationDto>> getFavorites(
+      @RequestHeader("Authorization") String header, Principal principal) {
+    UserInfoDto userInfo = userService.getUserInfo(header, principal);
+    List<Favorite> favorites = favoriteRepository.findByUserUuid(UUID.fromString(userInfo.getId()));
+    List<LocationResultLocationDto> response = favorites.stream()
+        .map(favorite -> {
+          Location location = favorite.getLocation();
+          return locationMapper.mapToOutputDto(location);
+        })
```

Comment:
Du kannst das auch schön in eine Zeile schreiben :D
`.map(favorite -> locationMapper.mapToOutputDto(favorite.getLocation()))`

Reply:
Na meinetwegen :D